packages/react-components/react-datepicker-compat/src/index.ts

```diff
@@ -1,7 +1,7 @@
-export { AnimationDirection, Calendar, useCalendarStyles_unstable } from './Calendar';
+export { calendarClassNames, AnimationDirection, Calendar, useCalendarStyles_unstable } from './Calendar';
 export type { CalendarNavigationIcons, CalendarProps, CalendarStyleProps, CalendarStyles, ICalendar } from './Calendar';
 
-export { CalendarDay, useCalendarDayStyles_unstable } from './CalendarDay';
+export { calendarDayClassNames, CalendarDay, useCalendarDayStyles_unstable } from './CalendarDay';
```

Comment:
_nit:_ Alphabetically order these

```suggestion
export { CalendarDay, calendarDayClassNames, useCalendarDayStyles_unstable } from './CalendarDay';
```